_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+Here are some of the findings:
+
+- The quality of governance varied widely from project to project. In four of the seven projects we found that governance responsibilities were not carried out adequately because key issues that impacted project performance were either not reported or not resolved.
+- Five of the seven projects we looked at were allowed to proceed with a business case that was incomplete or out-of-date or contained information that could not be supported.
+- Four of the projects were undertaken by departments that lacked the appropriate skills and experience to manage the projects or the capacity to use the system to improve the way they deliver their programs.
+- Depending on the project, the quality of project management ranged from good to seriously flawed. In two cases, poor project management led to long delays and large cost overruns.
+
+[2011 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_201106_02_e_35370.html): Examination of the progress made from the 2006 recommendations
+
+Here are some of the findings:
+
+- Overall, the government has made unsatisfactory progress on its commitments in response to our 2006 recommendations.Although some improvements have been made, progress has been unsatisfactory in the important areas of governance and project business cases. Only two of the five projects we looked at, the Temporary Resident Biometrics Project and Integrated Revenue Collections systems, met most of our criteria for well-managed projects.
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines.In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured.For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
+
+Unfortunately, after much effort, these decade old findings are still prevalent today. The underlying issue is slow response to change, and a large lead time to be able to incorporate them into projects and systems. One of the culprits of this is a complex release process which tends to promote large releases due to the efforts needed to go through it. [Large releases are risky](https://sencanada.ca/content/sen/committee/421/NFFN/reports/NFFN_Phoenix_Report_32_WEB_e.pdf). Each release involves many different stakeholders with distinct functional roles, and manual checks and handoffs. If you're looking at a diagram of this process for the first time, it's almost impossible to comprehend. It is difficult for one person to completely describe all the stages that it involves.
```

Comment:
Would 'cause' or 'lead to' be more clear than 'promote'?  in 'One of the culprits of this is a complex release process which tends to ...'.

Would 'as well as manual checks and handoffs' be better than 'and manual checks and handoffs'?